core/src/dird/authenticate_console.cc

```diff
@@ -424,7 +406,21 @@ bool AuthenticateConsole(UaContext* ua)
       LogErrorMessage(console_authenticator->console_name_, ua);
       return false;
     }
+    if (num_leases > me->MaxConsoleConnections) {
+      Emsg0(M_INFO, 0,
+            _("Number of console connections exceeded\n"
+              "Maximum: %u, Current: %u\n"),
+            me->MaxConsoleConnections, num_leases);
```

Comment:
```suggestion
            me->MaxConsoleConnections, static_cast<unsigned>(num_leases));
```
Sadly there is no `saturating_cast` but I do not foresee anybody having the limit close to the 32bit limit so that should not matter.

Reply:
If `Emsg` supports it one could also just use `%zu` instead.